tsconfig-common.json

```diff
@@ -24,6 +24,16 @@
         // "noImplicitAny": true,
         /* Completeness */
         "skipDefaultLibCheck": true, // it's safe to assume that built-in .d.ts files are correct
-        "skipLibCheck": false
+        "skipLibCheck": false,
+
+        "plugins": [
+            {
+                "name": "typescript-eslint-language-service"
+            },
+            {
+                "name": "ts-plugin-type-coverage",
+                "ignoreCatch": true,
+            }
```

Comment:
There are a bunch of options that we can experiment, see https://github.com/plantain-00/type-coverage#config-in-packagejson.  TS doesn't support type annotations on the `e` in `catch (e)` which is why I've decided to start with `ignoreCatch: true`.  The tool also provides a special comments to ignore type coverage on certain parts of code, see https://github.com/plantain-00/type-coverage#ignore-line.